src/UseCases/AddDonation/AddDonationUseCase.php

```diff
@@ -91,11 +97,32 @@ private function newTrackingInfoFromRequest( AddDonationRequest $request ): Trac
 		return $trackingInfo->freeze()->assertNoNullFields();
 	}
 
-	private function sendDonationConfirmationEmail( Donation $donation ) {
+	private function sendDonationConfirmationEmail( int $donationId, Donation $donation, bool $needsModeration ) {
 		if ( $donation->getPersonalInfo() !== null ) {
 			$this->mailer->sendMail(
 				new MailAddress( $donation->getPersonalInfo()->getEmailAddress() ),
-				[] // TODO
+				[
+					'recipient' => [
+						'salutation' => ( new GreetingGenerator() )->createGreeting(
+							$donation->getPersonalInfo()->getPersonName()->getLastName(),
+							$donation->getPersonalInfo()->getPersonName()->getSalutation(),
+							$donation->getPersonalInfo()->getPersonName()->getTitle()
+						)
+					],
+					'donation' => [
+						'id' => $donationId,
+						'formattedAmount' => ( new \NumberFormatter( 'de_DE', \NumberFormatter::CURRENCY ) )->format(
+							$donation->getAmount()
+						),
+						'needsModeration' => $needsModeration,
+						'paymentType' => [
+							'code' => $donation->getPaymentType(),
+							'text' => 'Überweisung' // TODO: use twig translator extension
+						],
+						'bankTransferCode' => 'W-Q-ABCDEF-G', // $donation->getBankTransferCode(), should this be fetched from entity?
```

Comment:
With my follow up to your other PR, you can simply get it from the domain object.